components/dashboard/public/manifest.json

```diff
@@ -1,19 +1,19 @@
 {
-  "short_name": "Gitpod",
```

Comment:
Can't change this hardcoded. Have to come from banding YAML or method to configure via build script.